app/src/main/AndroidManifest.xml

```diff
@@ -56,7 +56,8 @@
         <activity android:configChanges="orientation|screenSize"
                   android:launchMode="singleTask"
                   android:name="org.mifos.mobile.ui.activities.PassCodeActivity"
-                  android:windowSoftInputMode="adjustResize"/>
```

Comment:
Why is this `screenOrientation` added. Please clarify

Reply:
If the orientation is landscape in the passcode activity, the UI elements go out of bounds. mifos-mobile-cn also had a similar [problem](https://github.com/openMF/mifos-mobile-cn/issues/108) which was solved by restricting the orientation to only portrait.